docs/vite.config.js

```diff
@@ -25,7 +26,8 @@ const config = {
     }
   },
   define: {
-    'import.meta.env.VERCEL': JSON.stringify(process.env.VERCEL)
+    'import.meta.env.VERCEL': JSON.stringify(process.env.VERCEL),
+    'process.env.NODE_DEBUG': JSON.stringify('')
```

Comment:
Do we need this line?

Reply:
Yes we do. Otherwise errors are being thrown that it doesn't see it in env